src/network/core/game_info.h

```diff
@@ -76,16 +68,24 @@ struct NetworkGameInfo : NetworkServerGameInfo {
 	char server_name[NETWORK_NAME_LENGTH];          ///< Server name
 	char server_revision[NETWORK_REVISION_LENGTH];  ///< The version number the server is using (e.g.: 'r304' or 0.5.0)
 	bool dedicated;                                 ///< Is this a dedicated server?
-	bool version_compatible;                        ///< Can we connect to this server or not? (based on server_revision)
-	bool compatible;                                ///< Can we connect to this server or not? (based on server_revision _and_ grf_match
 	bool use_password;                              ///< Is this server passworded?
-	byte game_info_version;                         ///< Version of the game info
+	byte clients_on;                                ///< Current count of clients on server
 	byte clients_max;                               ///< Max clients allowed on server
 	byte companies_on;                              ///< How many started companies do we have
 	byte companies_max;                             ///< Max companies allowed on server
 	byte spectators_on;                             ///< How many spectators do we have?
 	byte spectators_max;                            ///< Max spectators allowed on server
-	byte map_set;                                   ///< Graphical set
+	byte landscape;                                 ///< The used landscape
+};
+
+/**
+ * The game information that is sent from the server to the clients
+ * with extra information only required at the client side.
+ */
+struct NetworkGameInfo : NetworkServerGameInfo {
+	bool version_compatible;                        ///< Can we connect to this server or not? (based on server_revision)
+	bool compatible;                                ///< Can we connect to this server or not? (based on server_revision _and_ grf_match
+	byte game_info_version;                         ///< Version of the game info
```

Comment:
While at it, we might as well throw away this variable; it is never used (well, yes, locally when reading GameInfo, but nothing after that).